### ISSUE TITLE
Amazon shopping compatibility report

### ISSUE THREAD

**Issue by jo45** [AdAway/AdAway#2488]
App Name: Amazon Shopping
Version: latest both for app and AdAway

App doesn't work because of DNS block

**Workaround**
Add fls-eu.amazon.fr to authorized hosts

`workaround-available`

**Comment by deepdvd** [comment]
Amazon app works fine for me unless I have the "Enable web server" turned on.  If I do that, I get the "UH-OH Something went wrong on our end" with the dog pictures.  Alternate workaround is to just disable the web server.

**Comment by mind-overflow** [comment]
Yep, same for me. It used to work fine with the webserver, but today after resetting my phone and installing AdAway 5.3.0, Amazon doesn't work unless i disable it.

**Comment by Aaron1011** [comment]
Would it be possible to add this by default? I spent quite a while trying too figure out why the Amazon login screen kept closing, until I thought to check AdAway
